composer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "blesta/digitaloceanmodule",
```

Comment:
You should include yourself as an author. See https://getcomposer.org/doc/04-schema.md#authors